Assets/MixedRealityToolkit/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -327,11 +327,10 @@ Shader "Mixed Reality Toolkit/Standard"
                 float4 vertex : POSITION;
                 float2 uv : TEXCOORD0;
 #if defined(LIGHTMAP_ON)
-                float2 lightMapUV : TEXCOORD1;
-#endif
-#if defined (_VERTEX_EXTRUSION_SMOOTH_NORMALS)
-                fixed3 smoothNormal : TEXCOORD2;
+                float2 uv1 : TEXCOORD1;
```

Comment:
Add comments here for what uv1, uv2, uv3 mean

Reply:
I made the names more generic since they now can be mutliple things. But, I agree. I'll comment what they can be in different circumstances.

Reply:
yea or at least acknowledge that these are uv slots for special components, depending on the feature